impexp-config/src/main/java/org/citydb/config/project/exporter/CityJSONOptions.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * 3D City Database - The Open Source CityGML Database
+ * http://www.3dcitydb.org/
+ *
+ * Copyright 2013 - 2019
+ * Chair of Geoinformatics
+ * Technical University of Munich, Germany
+ * https://www.gis.bgu.tum.de/
+ *
+ * The 3D City Database is jointly developed with the following
+ * cooperation partners:
+ *
+ * virtualcitySYSTEMS GmbH, Berlin <http://www.virtualcitysystems.de/>
+ * M.O.S.S. Computer Grafik Systeme GmbH, Taufkirchen <http://www.moss.de/>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.citydb.config.project.exporter;
+
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlSchemaType;
+import javax.xml.bind.annotation.XmlType;
+
+@XmlType(name = "CityJSONExportOptionsType", propOrder = {})
+public class CityJSONOptions {
+    @XmlElement(defaultValue = "false")
```

Comment:
I disabled pretty print for CityJSON per default because it has a much bigger impact on the resulting file size compared to CityGML. For example, every single coordinate of every vertex in the `"vertices"` array is put on a new line (this is done by the Gson library in the background and thus out of our control). Therefore, I thought pretty print makes only sense if you only export few objects and mainly for debugging purposes but not for general exports.

But at least I can add a pretty print option for CityGML as well.

Reply:
Agree. it makes sence to disable the prettyPrint per default. Yes, adding pretty print option for CityGML export is not bad.

Reply:
Ok, added with a939aeb3f4f67746bb6680f0ea360e0c3a6ebd9f. Note that this changes the previous default behaviour where pretty printing was used for CityGML per default when doing a compressed export (gzip or zip).

Reply:
Thanks for adding this. Looks very goold.